Entity/PluginEnhancerAnura.php

```diff
@@ -0,0 +1,172 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: nbush
+ * Date: 10/18/18
+ * Time: 3:22 PM.
+ */
+
+namespace MauticPlugin\MauticEnhancerBundle\Entity;
+
+use Doctrine\ORM\Mapping\ClassMetadata;
+use Mautic\CoreBundle\Doctrine\Mapping\ClassMetadataBuilder;
+use Mautic\CoreBundle\Entity\CommonEntity;
+
+/**
+ * Class PluginEnhancerAnura.
+ */
+class PluginEnhancerAnura extends CommonEntity
+{
+    const TABLE_NAME = 'plugin_enhancer_anura';
+
+    /**
+     * @var int
+     */
+    protected $id;
+
+    /**
+     * @var \DateTime
+     */
+    protected $dateAdded;
+
+    /**
+     * @var string
+     */
+    protected $ipAddress;
+
+    /**
+     * @var string
+     */
+    protected $userAgent;
+
+    /**
+     * @var string
+     */
+    protected $result;
+
+    /**
+     * @param ClassMetadata $metadata
+     */
+    public static function loadMetadata(ClassMetadata $metadata)
+    {
+        $builder = new ClassMetadataBuilder($metadata);
+
+        $builder->setTable(MAUTIC_TABLE_PREFIX.self::TABLE_NAME)
```

Comment:
Shouldn't this be `MAUTIC_TABLE_PREFIX . '.' . self::TABLE_NAME` ?

Reply:
No. This is the table prefix, not the database name.

Reply:
You are correct. I was incorrect.